qutebrowser/mainwindow/mainwindow.py

```diff
@@ -348,6 +357,9 @@ def _on_config_changed(self, option):
             self._update_overlay_geometries()
         elif option == 'window.hide_decoration':
             self._set_decoration(config.val.window.hide_decoration)
+        elif option == 'bg_transparency':
```

Comment:
That option doesn't exist, you probably mean `colors.webpage.bg`

Reply:
Ok I modified it